docs/dev/event_time.md

```diff
@@ -213,10 +213,33 @@ arrive after the system's event time clock (as signaled by the watermarks) has a
 timestamp. See [Allowed Lateness]({{ site.baseurl }}/dev/stream/operators/windows.html#allowed-lateness) for more information on how to work
 with late elements in event time windows.
 
+## Idling sources
```

Comment:
Source can be marked as temporarily idle. Maybe we should also mentioned it `SourceFunction.SourceContext#markAsTemporarilyIdle` here.

Reply:
Good point.  Probably a good idea to include the here the description in [StreamStatus](https://ci.apache.org/projects/flink/flink-docs-release-1.5/api/java/org/apache/flink/streaming/runtime/streamstatus/StreamStatus.html).

Reply:
I have added a reference to both things